packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -482,25 +482,30 @@ module.exports = ({ model, modelKey, strapi }) => {
   }
 
   function search(params, populate) {
-    // Convert `params` object to filters compatible with Mongo.
-    const filters = modelUtils.convertParams(modelKey, params);
-
-    const $or = buildSearchOr(model, params._q);
-    if ($or.length === 0) return Promise.resolve([]);
-
-    return model
-      .find({ $or })
-      .sort(filters.sort)
-      .skip(filters.start)
-      .limit(filters.limit)
-      .populate(populate || defaultPopulate)
-      .then(results => results.map(result => (result ? result.toObject() : null)));
+    const populateOpt = populate || defaultPopulate;
+
+    const search = buildSearchOr(model, params._q);
```

Comment:
What do you think about moving the the buildSearchOr inside buildQuery as we can't make them independent like in bookshelf ?